process/OpenHW-Specification-Process.md

```diff
@@ -0,0 +1,170 @@
+# OpenHW Process Document: Specification Process, States and Format
```

Comment:
No process is complete without templates and tools. Recommend to provide a template github project from which to start. I would also question if the Specification Process shouldn't just use its own process, and thus its own template. In this case, you should move from MD to RST